pom.xml

```diff
@@ -409,6 +409,11 @@
             </signature>
           </configuration>
         </plugin>
+        <plugin>
+          <groupId>org.codehaus.mojo</groupId>
+          <artifactId>properties-maven-plugin</artifactId>
```

Comment:
we'll need a works-with CQ for this dependency ...

Reply:
@calohmn  did you create the CQ for this already?

Reply:
Yes, I've created it.